### DISCUSSION TITLE
atom-mermaid package

### DISCUSSION THREAD

**Discussion by tuttifruttihat** [pulsar-edit/.github#174] [Community Package Help]
Is it possible to render mermaid diagrams in Pulsar? The package **atom-mermaid** seems no longer working.

**Comment by Daeraxa** [comment]
What do you mean by no longer working? I just tried installing it and it is working fine for me with the example in the README gif.

**Reply by tuttifruttihat** [reply]
I received a warning saying it is "deprecated", but I'm glad to hear it is working. I guess probably I made something wrong. Thank you!
